docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,305 @@
-Neptune-PyTorch Lightning Integration
-=====================================
+PyTorch Lightning Integration
+=============================
 
-The integration enables you to log |PyTorch Lightning| metrics to Neptune.
+What will you get?
+------------------
+[video-placeholder]
 
-.. image:: ../_static/images/integrations/pytorchlightning_neptuneml.png
-   :target: ../_static/images/integrations/pytorchlightning_neptuneml.png
-   :alt: PyTorchLightning neptune.ai integration
+PyTorch Lightning is a lightweight PyTorch wrapper for high-performance AI research. With Neptune integration you can:
 
-Requirements
-------------
-This integration is enabled as part of PyTorch Ligthning loggers module, so all you need is to have |neptune-client| installed.
+* preview running experiment
```

Comment:
Not 100% that preview is correct here. 
Monitor live or see experiment as it is running is clear and simple. 

I am good with preview if we have users using this term -> just don't have that info yet.

Reply:
I'll use "see experiment as it is running"